stories/styled-static-object.tsx

```diff
@@ -28,3 +37,11 @@ export const ObjectLiteralMapWithKeys = (): JSX.Element => (
     ))}
   </div>
 );
+
+export const ObjectLiteralWithSelector = (): JSX.Element => {
+  return (
+    <BackgroundWithSelector>
+      <div id="id-selector">Green text in selector div</div>
```

Comment:
nit.
Can we change the text to "Green box in selector div". Otherwise, without knowing the actual code,  I would expect to see a green text on white background in the VR reference